client/luigi-client.js

```diff
@@ -109,10 +109,13 @@
         var hashRoutingModeActive =
```

Comment:
Please move this variable initialisation to the inner scope of the new `if` block you created.

Reply:
done